test/Microsoft.AspNetCore.Authentication.Test/Google/GoogleMiddlewareTests.cs

```diff
@@ -781,6 +789,13 @@ private static TestServer CreateServer(GoogleOptions options, Func<HttpContext,
                         {
                             await context.Authentication.ChallengeAsync("Google");
                         }
+                        else if (req.Path == new PathString("/tokens"))
+                        {
+                            var authContext = new AuthenticateContext(TestExtensions.CookieAuthenticationScheme);
```

Comment:
Internally we have a fairly widespread pattern of creating a context, and passing it to a method, I don't think its THAT bad.  But its not the best public API no :)